src/static/svg/devnet.svg

```diff
@@ -1,107 +1,19 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
why did you edit svgs wit networks' icons?

Reply:
I had a problem with these icons after cloning the project. I will fix it